x-pack/plugins/ml/server/models/job_validation/validate_cardinality.ts

```diff
@@ -52,15 +67,17 @@ const validateFactory = (callWithRequest, job) => {
 
     if (relevantDetectors.length > 0) {
       try {
-        const uniqueFieldNames = _.uniq(relevantDetectors.map(f => f[fieldName]));
+        const uniqueFieldNames: string[] = [
+          ...new Set(relevantDetectors.map(f => f[fieldName])),
+        ] as string[];
```

Comment:
nit, could be `const uniqueFieldNames = [...new Set(relevantDetectors.map(f => f[fieldName] as string))];`

Reply:
Done in https://github.com/elastic/kibana/pull/65971/commits/90ba7a4ca3e8ba42a52d1afcb76d79aaf5a161b2